python/utils/LinkedLibraryHolder.h

```diff
@@ -64,8 +64,14 @@ class LinkedLibraryHolder {
   /// @brief Map of available targets.
   std::unordered_map<std::string, RuntimeTarget> targets;
 
+  /// @brief Map of simulation targets
+  std::unordered_map<std::string, RuntimeTarget> simulationTargets;
+
+  /// @brief Store the name of the default target
+  std::string defaultTarget = "";
```

Comment:
`std::string` is default initialized to an empty string.
```suggestion
  std::string defaultTarget;
```